Resources/Views/accessory.leaf

```diff
@@ -52,6 +52,12 @@
                                 <p>This accessory requires a proprietary <a href="/accessories/#(accessory.id)">bridge</a>.</p>
                             </div>
                         }
+                        #if(accessory.supports_airplay_2) {
+                            <div id="accessory-page-feature">
+                                <img src="/images/airplay.png">
+                                <p>This accessory supports Airplay 2</p>
```

Comment:
The official name is AirPlay, with the capital P, so we should change it everywhere so users can better recognize it

Reply:
Fixed !